chain/chain.go

```diff
@@ -115,6 +108,21 @@ func (c *RPCClient) Start() error {
 		return errors.New("mismatched networks")
 	}
 
+	// Ensure the RPC server has a compatible API version.
+	var serverAPI semver
+	versionResult, err := c.Version()
+	if err == nil {
```

Comment:
Right it will just leave it as the zero value.  Since the required version is greater than 0.0.0 it won't pass.  The remote dcrd json-rpc server might be too old to have the version rpc so this is done to gracefully handle that error.